main/src/com/google/refine/operations/row/RowDuplicatesRemovalOperation.java

```diff
@@ -0,0 +1,130 @@
+/*
+
+Copyright 2024 OpenRefine
+All rights reserved.
```

Comment:
This copyright header seems different from the other ones we use, with the usual BSD Clause 3 license. Was it on purpose?